synapse/handlers/device.py

```diff
@@ -265,6 +265,14 @@ def __init__(self, hs):
 
         hs.get_distributor().observe("user_left_room", self.user_left_room)
 
+    def _check_device_name_length(self, name):
```

Comment:
```suggestion
    def _check_device_name_length(self, name: str):
```

Also a quick docstring would be nice as well, something like:

```python
"""Checks whether a device name is longer than the maximum allowed length.

Args:
    name: The name of the device.

Raises:
    SynapseError: if the device name is too long.
"""
```

Reply:
Makes sense, thnx!